README.md

```diff
@@ -138,19 +170,17 @@ Or you can toggle it inside IRB/Pry:
 >> MethodFinder.toggle_debug!
 ```
 
-## Warning
+## Development
 
-Common sense not included!
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-While I never had any problems with this, it's still better to be safe than
-sorry, so use this with caution and maybe not on production data.
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
-I initially wrote this for the students of the core Ruby course on
-[RubyLearning](http://rubylearning.org), so Rails is not of interest to me (not
-saying it doesn't work there, just that I test in plain IRB/Pry and not with
-the Rails console.
+## Contributing
 
-## Thanks
+Bug reports and pull requests are welcome on GitHub at https://github.com/citizen428/methodfinder.
+
+### Noteworthy contributors
```

Comment:
I can't speak for the other contributors, but I think a changelog would be better. Thanks/credits could be included there manually, or automatically via a changelog generator (I recommend [this one](https://github.com/github-changelog-generator/github-changelog-generator) ([alternatives](https://github.com/github-changelog-generator/github-changelog-generator/wiki/Alternatives))).

It would also be a good idea to add git tags for each release (that tool uses them). I might add them if you don't beat me to it :-)

Reply:
[Done](https://github.com/citizen428/methodfinder/releases).

These are mostly based on when the version changed in the [`.gemspec`](https://github.com/citizen428/methodfinder/commits/master/methodfinder.gemspec). If you know better, please update the tags. You'll need to delete and re-push each one e.g.:

```
$ git tag -d v1.2.3
$ git push --delete origin v1.2.3
$ git tag -a v1.2.3 -m v1.2.3 <commit>
$ git push --tags
```